conf/messages

```diff
@@ -778,14 +778,14 @@ toDate.year.label=Year
-declareCapacity.details.ownerOrOccupier.text=If you’re the owner or occupier for only part of the property, you can claim this property. Once your claim is approved, you may wish to request separate valuations for each part of the property.
-declareCapacity.details.notOwner=I’m not the owner but I let the property to someone else, as a sublet
-declareCapacity.details.notOwner.text=<p>If you’re not the owner of the property but you let all or part of the property to another person or business as a sublet, you should select ‘owner’ and enter the dates of the sublet period.</p><p>If you let only a part of the property to another person or business, you can claim this property. Once your claim is approved, you may wish to request separate valuations for each part of the property.</p>
-declareCapacity.details.ownerOrOccupier.client=My client is the owner or occupier for only part of this property
-declareCapacity.details.ownerOrOccupier.text.client=If your client is the owner or occupier for only part of the property, they can claim this property. Once the claim is approved, you may wish to request separate valuations for each part of the property.
-declareCapacity.details.notOwner.client=My client is not the owner but they let the property to someone else, as a sublet
-declareCapacity.details.notOwner.text.client=<p>If your client is not the owner of the property but lets all or part of the property to another person or business as a sublet, you should select ‘owner’ and enter the dates of the sublet period.</p><p>If your client lets only a part of the property to another person or business, they can claim this property. Once the claim is approved, you may wish to request separate valuations for each part of the property.</p>
+declareCapacity.details.ownerOrOccupier=I own or occupy a part of this property
+declareCapacity.details.ownerOrOccupier.text=After the Valuation Office Agency have approved your connection, you may want to ask them to split this property to get separate valuations for each part.
+declareCapacity.details.notOwner=I do not own the property but I sublet it to someone else
+declareCapacity.details.notOwner.text=<p>If you sublet the property to another person or business, select ‘Owner’ and enter the dates of the sublet period.</p><p>If you only sublet part of the property, you may want to ask the Valuation Office Agency to split this property to get separate valuations for each part. You can do this after they have approved your connection to the property.</p>
+declareCapacity.details.ownerOrOccupier.client=My client owns or occupies part of this property
+declareCapacity.details.ownerOrOccupier.text.client=<p>You can still add this property on behalf of your client, as if they owned or occupied the whole property.</p><p>After we’ve approved your client’s connection to the property, you can ask us to split the property to get separate valuations for each part.</p>
```

Comment:
are you missing `class="govuk-body"` on your p tags

Reply:
This class should be added directly in messages? This <p>'s are wrapped in div "govuk-details_text" as a part of govukDetails component.

Reply:
This p's are wrapped in div "govuk-details_text" as a part of govukDetails component.*

Reply:
i personally don't like html in messages file. I know that there are lots of others here as well (may be we'll handle them as a separate task). Are you saying that the class is not needed because the styles will be inherited from "govuk-details_text"? So long as you've tested this and that is the case then im happy 👍

Reply:
Yes. I did test it. This solution was present in code before. I just swap text in <p>. Thanks for comment.